### ISSUE TITLE
Mediaman isn't installing

### ISSUE THREAD

**Issue by webcrawlersites** [FarhanShares/laravel-mediaman#85]
I try to run the command `composer require farhanshares/laravel-mediaman`, but it gives me the following error:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - farhanshares/laravel-mediaman[v0.1.0, ..., v0.5.1] require illuminate/support ^7.0|^8.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - farhanshares/laravel-mediaman v0.6.0 requires illuminate/support ^7.0|^8.0|^9.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - farhanshares/laravel-mediaman[v0.7.0, ..., v0.9.0] require illuminate/support ^7.0|^8.0|^9.0|^10.0 -> found illuminate/support[v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.22] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires farhanshares/laravel-mediaman * -> satisfiable by farhanshares/laravel-mediaman[v0.1.0, ..., v0.9.0].

You can also try re-running composer require with an explicit version constraint, e.g. "composer require farhanshares/laravel-mediaman:*" to figure out if any version is installable, or "composer require farhanshares/laravel-mediaman:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

I also ran `composer update` and then retried the install command, but it gives me the same error message.

Here are the specficis I am running:
Laravel Version: 11.23.5
PHP Version: 8.2.13

**Comment by FarhanShares** [comment]
Thanks for reporting the issue @webcrawlersites, I'll have a look into it soon.

**Comment by FarhanShares** [comment]
@webcrawlersites 
The issue has been fixed by #86 

Closing it, feel free to re-open if needed.
